tests/ui/sanitizer/cfi-closures.rs

```diff
@@ -0,0 +1,45 @@
+// Check various forms of dynamic closure calls
```

Comment:
This is curiously missing any tests with higher-ranked lifetimes. Please add some.